frontend/public/scss/catalog.scss

```diff
@@ -79,379 +78,359 @@
+        @include media-breakpoint-down(sm) {
+            padding: 20px 0px;
+          }
+        h2 {
+            overflow: hidden;
+            color: #FFF;
+            text-align: center;
+            text-overflow: ellipsis;
+            /* H3 */
+            font-size: 28px;
+            font-style: normal;
+            font-weight: 700;
+            line-height: normal;
+            display: flex;
             flex-direction: column;
-            align-items: flex-start;
-            gap: 2px;
-            flex: 1 0 0;
-            z-index: 10;
-        }
-    }
-
-    .container-fluid {
-        margin-right: 0;
-        margin-left: 0;
-        padding-left: 0;
-        padding-right: 0;
-        width: 100%;
-        max-width: revert;
-
-        .row {
-            margin-right: 0;
+            justify-content: center;
+            margin-bottom: 0;
         }
-
-        .lds-ring {
-            position: relative;
-            width: 80px;
-            height: 80px;
-            margin: auto;
-            padding: 8px;
-          }
-          .lds-ring div {
-            box-sizing: border-box;
-            display: block;
-            position: absolute;
-            width: 64px;
-            height: 64px;
-            margin: 8px;
-            border: 8px solid #fff;
-            border-radius: 50%;
-            animation: lds-ring 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
-            border-color: #fff transparent transparent transparent;
-          }
-          .lds-ring div:nth-child(1) {
-            animation-delay: -0.45s;
-          }
-          .lds-ring div:nth-child(2) {
-            animation-delay: -0.3s;
-          }
-          .lds-ring div:nth-child(3) {
-            animation-delay: -0.15s;
-          }
-          @keyframes lds-ring {
-            0% {
-              transform: rotate(0deg);
-            }
-            100% {
-              transform: rotate(360deg);
+    
+        #mobile-catalog-title {
+            button {
+                width: 24px;
+                height: 24px;
+                position: absolute;
+                left: 20px;
+                background: url("#{$static-path}/images/filter_list_alt.svg") no-repeat;
             }
-          }
+        }
     }
 
-    #tab-row {
-        height: 70px;
-        @include media-breakpoint-up(md) {
-            margin-left: 0;
-        }
+    .container {
+        max-width: 1320px;
 
-        .col {
+        @include media-breakpoint-down(md) {
             padding-left: 0;
-            padding-right: 0;
         }
-
-        .catalog-count-animation {
-
-            /* starting ENTER animation */
-            .count-enter {
-                right: 0;
-                width: inherit;
-                position: absolute;
-                margin-right: 64px;
-                opacity: 0;
-            }
+    
+        #course-catalog-navigation {
+            display: flex;
+            margin: 0 auto;
+            font-family: 'Inter';
```

Comment:
I add the font fallbacks in just in case - if google fonts is down, they have fonts turned off, etc - just to make sure we get the san serif default at least. This also helps with folks with special fonts turned on for accessibility reasons. In my browser, it shows the full font-family prior crossed out, and only "inter" for this which leads me to believe it might not have the fallback family in place.